.github/workflows/pr_integration_tests.yml

```diff
@@ -95,6 +99,10 @@ jobs:
         run: |
           docker pull vishnunair/docker-redis-cluster:latest
           docker run -d -p 6001:6379 -p 6002:6380 -p 6003:6381 -p 6004:6382 -p 6005:6383 -p 6006:6384 --name redis-cluster vishnunair/docker-redis-cluster
+      - name: Setup Milvus Cluster
+        run: |
+          wget https://github.com/milvus-io/milvus/releases/download/v2.4.20/milvus-standalone-docker-compose.yml -O docker-compose.yml
```

Comment:
WDYT about using latest release version -> https://github.com/milvus-io/milvus/releases/tag/v2.5.2

Reply:
i tested 2.4.20 locally but i can test the latest

Reply:
Based on the docs, the latest available image is 2.5.1

https://milvus.io/docs/install_standalone-docker-compose.md#Run-Milvus-with-Docker-Compose-Linux

I tested it and it works. I'll update.